cmd/utils/flags.go

```diff
@@ -457,6 +457,10 @@ var (
 		Usage: "The number of blocks should be persisted in db (default = 86400)",
 		Value: uint64(86400),
 	}
+	PruneAncientDataFlag = cli.BoolFlag{
+		Name:  "pruneancient",
+		Usage: "Keep a fixed number of blocks(9w) in kvstore under full syncmode, enable release more os resources",
```

Comment:
Please emphasize who is suggested to enable this feature, like anyone who don't care about the history blocks.

Reply:
And warn user the risk to enable this feature, like they can not revert this setting, they can not recover history data by disable this feature.